### PR TITLE
DAT-2866 citation error wont be showing up when partial parse is run

### PR DESCRIPTION
**Pull request by idradm**

### CODE REVIEW

**Comment by SebastianMarzjan** [comment]
LGTM
@macbre ? :)

File: extensions/Cite/Cite_body.php [hunk 1]
```diff
@@ -1070,7 +1070,7 @@ function checkRefsNoReferences( &$parser, &$text ) {
 			return $parser->extCite->checkRefsNoReferences( $parser, $text );
 		}
 
-		if ( $parser->getOptions()->getIsSectionPreview() ) {
+		if ( $parser->getOptions()->getIsSectionPreview() || $parser->getOptions()->getIsPartialParse() ) {
```
Comment (macbre) [inline extensions/Cite/Cite_body.php]: Please wrap it in `Wikia change` block

**Comment by macbre** [comment]
Had an offline code review with @idradm.

LGTM given the fact that `Parser::internalParse` does not suite you needs here. A minor comment left.

**Comment by idradm** [comment]
Merging as it'll go through second CR in parent branch

**Comment by macbre** [comment]
LGTM, thanks for the unit tests!

**Pull request merged**